template/pages/MyEvents.html

```diff
@@ -0,0 +1,549 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <title></title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, shrink-to-fit=no, initial-scale=1">
+    <link href="../resources/styles/bootstrap.css" rel="stylesheet" />
+    <script src="../resources/scripts/jquery.js"></script>
+    <script src="../resources/scripts/popper.min.js"></script>
+    <script src="../resources/scripts/bootstrap.js"></script>
+    <style>
```

Comment:
Can we create a separate file for the styles (in all affected areas) rather than mixing it with the html.